setup.sh

```diff
@@ -43,13 +43,13 @@ if [[ $BVER =~ $VERSION_REGEX ]] ; then
 elif [ $BROWSER == "safari" ] && [ $BVER == "unstable" ]; then
   # This is quite dangerous, it is scraping the safari download website for the URL. If the format
   # of the website changes then it won't work anymore. We should add safari to
-  # browsers.contralis.info instead
+  # browsers.contralis.info / browser-sleuth instead
   TARGET_URL=`curl https://developer.apple.com/safari/download/ | sed -nE 's/.*href="(.*\.dmg)">.*macOS 10.12.*/\1/p'`
   TARGET_VERSION=`curl https://developer.apple.com/safari/download/ | sed -nE 's/.*>([0-9]+)<\/p>.*$/\1/p'`
 else
-  TARGET_BROWSER=`curl -H 'Accept: text/csv' http://browsers.contralis.info/$PLATFORM/$BROWSER/$BVER`
-  TARGET_URL=`echo $TARGET_BROWSER | cut -d',' -f7`
-  TARGET_VERSION=`echo $TARGET_BROWSER | cut -d',' -f5`
+  TARGET_BROWSER=`node_modules/browser-sleuth/sleuth.sh $BROWSER $BVER $PLATFORM`
```

Comment:
This causes an issue with the way I use travis-multirunner. I do something like

```
    cd node_modules/travis-multirunner
    if [ "${TRAVIS_OS_NAME}" == 'linux' ]; then
      BROWSER=chrome ./setup.sh
      BROWSER=firefox ./setup.sh
      export CHROME_BIN=$(pwd)/browsers/bin/chrome-$BVER
      export FIREFOX_BIN=$(pwd)/browsers/bin/firefox-$BVER
    else
      BROWSER=safari ./setup.sh
    fi
```

which gives the following error:

```
++node_modules/browser-sleuth/sleuth.sh chrome beta linux
./setup.sh: line 50: node_modules/browser-sleuth/sleuth.sh: No such file or directory
+TARGET_BROWSER=
```

Reply:
I think you could replace the hardcoded path with something like

```bash
TARGET_BROWSER=`node -e 'console.log(require.resolve("browser-sleuth/sleuth.sh")) $BROWSER $BVER $PLATFORM`
```

and the usage I described would continue to work.

Reply:
or alternatively write a little script inside travis-multirunner that simplifies the split even

Reply:
went your route